aries-site/src/pages/templates/wizard.mdx

```diff
@@ -85,3 +86,16 @@ When additional guidance is necessary, use a two-column wizard. Click "See Fulls
 >
   <TwoColumnWizardExample />
 </Example>
+
+### With validation
+
+Validation should occur on each step. If a user has not properly completed a given step, they should not be able to advance to subsequent steps.
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/wizard/TwoColumnWizardExample.js"
+  figma="https://www.figma.com/file/GEhgYzx3b8MejBjg9gAxyA/Wizarding?node-id=71%3A406"
```

Comment:
Should we exclude the Figma link here since it is not pointing to any specific validation example?

Reply:
Yes, I think you're right. I will remove it.